assets/helpers/externalLinks.js

```diff
@@ -256,7 +256,9 @@ function getDigitalCheckout(
   params.set('countryGroup', countryGroups[cgId].supportInternationalisationId);
   params.set('startTrialButton', referringCta || '');
 
-  return `${subsUrl}/checkout?${params.toString()}`;
+  // TODO!!
+  // return `${subsUrl}/checkout?${params.toString()}`;
+  return `/${countryGroups[cgId].supportInternationalisationId.toLowerCase()}/subscribe/digital/checkout?${params.toString()}`;
```

Comment:
Will this link the digital landing page to the new checkout?

Reply:
This should be removed. When I did this it was because I didn't have any `subsUrl` other than `subscribe.thegu...`